dotnet/samples/KernelSyntaxExamples/Example23_OpenAPIPlugin.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Plugins.OpenApi;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Examples;
+
+/// <summary>
+/// Examples to show how to create plugins from OpenAPI specs.
+/// </summary>
+public class Example23_OpenAPIPlugin : BaseTest
+{
+    /// <summary>
+    /// Example to show how to consume operation extensions and other metadata from an OpenAPI spec.
+    /// Try modifying the sample schema to simulate the other cases by
+    /// 1. Changing the value of x-openai-isConsequential to true and see how the function execution is skipped.
+    /// 2. Removing the x-openai-isConsequential property and see how the function execution is skipped.
+    /// </summary>
+    [Fact]
+    public async Task RunOpenAIPluginWithMetadataAsync()
+    {
+        Kernel kernel = new();
+
+        // This HTTP client is optional. SK will fallback to a default internal one if omitted.
+        using HttpClient httpClient = new();
+
+        // Create a sample OpenAPI schema that calls the github versions api, and has an operation extension property.
+        // The x-openai-isConsequential property is the operation extension property.
+        var schema =
+            "{" +
+            "  \"openapi\": \"3.0.1\"," +
+            "  \"info\": {" +
```

Comment:
With recent upgrades of C# language version in repo, you should be able to use raw string literals, which should simplify usage and representation of JSON in code. For example:
https://github.com/microsoft/semantic-kernel/blob/c72080d7da6baa539eab515fc065730d87d191e2/dotnet/src/Connectors/Connectors.UnitTests/OpenAI/ChatCompletion/OpenAIChatCompletionServiceTests.cs#L352-L356

Reply:
Oh yeah, that's much better. Will fix, thanks!